bin/common.bash

```diff
@@ -106,7 +106,7 @@ append_trap() {
 
 # Write PGP fingerprints to SOPS config
 sops_config_write_fingerprints() {
-    yq n 'creation_rules[0].pgp' "${1}" > "${sops_config}" || \
+    yq e -n '{"creation_rules[0].pgp" : "'${1}'"}' > "${sops_config}" || \
```

Comment:
We are looking to migrate 4.26 and we don't need to specify eval option after 4.20
```suggestion
    yq -n '{"creation_rules[0].pgp" : "'${1}'"}' > "${sops_config}" || \
```